Rename urban nature supply and add accessible urban nature

phargogh opened the pull request:
This PR does two things:
1. Urban nature supply has been renamed urban nature supply per capita
2. A new output file has been added for accessible urban nature

This will require an update to the UG, but I haven't done that yet.  It seems simpler to wait to do the update until the changes here are merged into `main`.

Fixes #1344

## Checklist
- [x] Updated HISTORY.rst and link to any relevant issue (if these changes are user-facing)
~~- [ ] Updated the user's guide (if needed)~~
~~- [ ] Tested the affected models' UIs (if relevant)~~

davemfish reviewed (changes requested):
Hey @phargogh this looks good to me. I have one request about some repetitive code in the tests. And I see the RTD builds are complaining about some indentation in HISTORY.rst. It's not immediately obvious where that problem is. Thanks!

phargogh commented on tests/test_urban_nature_access.py:
```diff
@@ -532,6 +542,34 @@ def test_split_urban_nature(self):
         admin_vector = None
         admin_layer = None
 
+        def _assert_urban_nature(path, sum_value, min_value, max_value):
```
Patched in https://github.com/natcap/invest/pull/1349/commits/193c6beaa7fb78e9bc6572d534beba5a7ca6103c

phargogh commented:
Thanks @davemfish !  I fixed up the RST issue (and found a new RST linter, `doc8`, `mamba install doc8`) and updated the duplicated method so that it's a class method.
